src/domexpression.js

```diff
@@ -31,6 +37,21 @@ var _ = Mavo.DOMExpression = $.Class({
 			this.expression = expression;
 		}
 
+		if (this.attribute?.startsWith("mv-attr-")) {
+			this.originalAttribute = this.attribute;
+			this.attribute = this.attribute.replace("mv-attr-", "");
+
+			if ([SVG_NAMESPACE_URI, MATHML_NAMESPACE_URI].includes(this.element.namespaceURI)) {
+				this.attribute = Mavo.getProperAttributeCase(this.element, this.attribute);
```

Comment:
Just dawned on me: do we also need to do this in XHTML documents (with an actual XML MIME type)? If so, it would require a different check.

Reply:
Probably we should. Our users will push us to do this in any case. 😅